pkg/recipes/terraform/config/providers/types.go

```diff
@@ -34,13 +34,37 @@ type Provider interface {
 	BuildConfig(ctx context.Context, envConfig *recipes.Configuration) (map[string]any, error)
 }
 
-// GetSupportedTerraformProviders returns a map of Terraform provider names to provider config builder.
-// Providers represent Terraform providers for which Radius generates custom provider configurations.
-// For example, the Azure subscription id is added to Azure provider config using Radius Environment's Azure provider scope.
-func GetSupportedTerraformProviders(ucpConn sdk.Connection, secretProvider *ucp_provider.SecretProvider) map[string]Provider {
+// GetUCPConfiguredTerraformProviders returns a map of Terraform provider names with configuration details stored in UCP, to provider config builder.
```

Comment:
UCP part is covered in the next sentence, this update I think is unclear..

```suggestion
// GetUCPConfiguredTerraformProviders returns a map of Terraform provider names to provider config builder.
```

Reply:
updated